package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "isdcf-registries",
-  "version": "1.0.0-beta.4",
+  "version": "1.0.0-beta.5",
```

Comment:
What is our policy re: revving version?

Reply:
[semver](https://semver.org) I'd imagine? This particular change would most correctly be a PATCH-level change, since it's just a bug fix, no new functionality.

Not sure how you handle the fact that we're still using the `beta` suffix.

Reply:
>  This particular change would most correctly be a PATCH-level change, since it's just a bug fix, no new functionality.

So, in this particular case:

- the version is incremented on every release, which is a signal that users should adopt the software
- the issue is severe enough that a new release is mandated
- therefore the version is incremented

Did I get this right?

> Not sure how you handle the fact that we're still using the beta suffix.

We should move to 1.0.0

I will create a separate issue.

Reply:
> Did I get this right?

Yes exactly.

Reply:
@radford-for-smpte What about using this opportunity to release 1.0.0, and closing #59 ?

Reply:
> 1.0.0

sounds like a great idea.

Reply:
Ok. Maybe merge this into master to make sure everything works, and then immediately create a new commit for 1.0.0?